client/containers/Pub/SpubHeader/SpubHeaderToolBar/spubHeaderToolbar.scss

```diff
@@ -0,0 +1,40 @@
+@import 'styles/variables.scss';
+
+.spubheader-toolbar {
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
```

Comment:
I think you actually want `align-items: center` here to vertically center the button in the toolbar.